### PR TITLE
add kurdish (sorani) language

### PR DESCRIPTION
**Pull request by swara-mohammed**

### CODE REVIEW

**Review by kylekatarnls — APPROVED** [review]

**Review by jwtiyar — APPROVED** [review]
Seems correct to me.

**Review by roj1512 — APPROVED** [review]
LGTM

**Comment by kylekatarnls** [comment]
Hello, I generated a test file for this new locale:
https://github.com/briannesbitt/Carbon/pull/2571/files#diff-a954a82a5b73e9e252be75f4fbdd4418ec4211cf8e2a2d10d1760dcee203b6e3

Please check that output would be globally fine for the given code snippet (in comment above each result line).

**Comment by swara-mohammed** [comment]
> Hello, I generated a test file for this new locale: https://github.com/briannesbitt/Carbon/pull/2571/files#diff-a954a82a5b73e9e252be75f4fbdd4418ec4211cf8e2a2d10d1760dcee203b6e3
> 
> Please check that output would be globally fine for the given code snippet (in comment above each result line).

thanks for the test. I noticed a few wrong translations that I will fix.

File: src/Carbon/Lang/ckb.php [hunk 1]
```diff
@@ -29,20 +29,20 @@
...
+    'a_week' => implode('|', ['{0}:count هەفتەیەک', '{1}هەفتە', '{2}هەفتە', ']2,11[:count هەفتە', ']10,Inf[:count هەفتە']),
+    'day' => implode('|', ['{0}:count ڕۆژێک', '{1}ڕۆژ', '{2}ڕۆژ', ']2,11[:count ڕۆژ', ']10,Inf[:count ڕۆژ']),
+    'a_day' => implode('|', ['{0}:count ڕۆژێک', '{1}ڕۆژ', '{2}ڕۆژ', ']2,11[:count ڕۆژ', ']10,Inf[:count ڕۆژ']),
+    'hour' => implode('|', ['{0}:count کاتژمێرێک', '{1}کاتژمێر', '{2}کاتژمێر', ']2,11[:count کاتژمێر', ']10,Inf[:count کاتژمێر']),
+    'a_hour' => implode('|', ['{0}:count کاتژمێرێک', '{1}کاتژمێر', '{2}کاتژمێر', ']2,11[:count کاتژمێر', ']10,Inf[:count کاتژمێر']),
+    'minute' => implode('|', ['{0}:count خولەکێک', '{1}خولەک', '{2}خولەک', ']2,11[:count خولەک', ']10,Inf[:count خولەک']),
+    'a_minute' => implode('|', ['{0}:count خولەکێک', '{1}خولەک', '{2}خولەک', ']2,11[:count خولەک', ']10,Inf[:count خولەک']),
+    'second' => implode('|', ['{0}:count چرکەیەک', '{1}چرکە', '{2}چرکە', ']2,11[:count چرکە', ']10,Inf[:count چرکە']),
+    'a_second' => implode('|', ['{0}:count چرکەیەک', '{1}چرکە', '{2}چرکە', ']2,11[:count چرکە', ']10,Inf[:count چرکە']),
```
Comment (kylekatarnls) [inline src/Carbon/Lang/ckb.php:45]: I guess this needs to be different either it's in a context of ago/from_now/before/after or for a simple duration.

Like we have the year_ago, month_ago etc. in https://github.com/briannesbitt/Carbon/blob/2fd2c4a77d58a4e95234c8a61c5df1f157a91bf4/src/Carbon/Lang/sk.php

Or declensions for ago/from_now/before in
https://github.com/briannesbitt/Carbon/blob/2fd2c4a77d58a4e95234c8a61c5df1f157a91bf4/src/Carbon/Lang/lt.php

`year` and `a_year` are the default such as in:
`Duration: 2 years`

`year_ago` will be used in:
`2 years ago`

`year_from_now` will be used in:
`2 years from now`

`year_before` will be used in:
`2 years before an other date`

`year_after` will be used in:
`2 years after an other date`

Comment (kylekatarnls) [reply src/Carbon/Lang/ckb.php:45]: I'm also a bit surprised. It's different only for `2`? `1`, `3`, `20` does not vary?

Comment (swara-mohammed) [reply src/Carbon/Lang/ckb.php:45]: If you mean, why don't names change after numbers, then in Kurdish (Sorani) the names will not be plural after numbers, but the number will show that plural.  It is correct, don't worry.

Comment (kylekatarnls) [reply src/Carbon/Lang/ckb.php:45]: OK for this about pluralization, what about the change, what's different between ڕۆژ and ڕۆژێک

Will the last version be good in any context: duration/ago/from_now/before/after?

Comment (swara-mohammed) [reply src/Carbon/Lang/ckb.php:45]: the mean of ڕۆژ is: day, but ڕۆژێک is: a day.

**Comment by roj1512** [comment]
Awesomely done. @swara-mohammed! Thank you too, @kylekatarnls!

**Pull request merged**